frame/App/Body/Main/Layout/ListTodo/index.js

```diff
@@ -33,9 +36,14 @@ const ListTodo = ({
         />
         show completed
       </div>
-      <AddItem
-        listId={id}
-      />
+      <Box>
+        <button type="button" onClick={() => { setOpenModal(!openModal); }}>Add New List Item</button>
+        <Modal open={openModal}>
+          <AddItem
+            listId={id}
+          />
+        </Modal>
+      </Box>
```

Comment:
You can simplify your life without the box right?
```suggestion
      <button type="button" onClick={() => { setOpenModal(!openModal); }}>Add New List Item</button>
      <Modal open={openModal}>
        <AddItem
          listId={id}
        />
      </Modal>
```